packages/notifi-dapp-example/src/components/AlertSubscriptionBlock.tsx

```diff
@@ -14,7 +14,7 @@ export const AlertSubscriptionBlock: React.FC<AlertSubscriptionRowProps> = ({
       <div className=" bg-white text-center py-3 font-medium border border-b-gray-200 rounded-md text-lg">
         {labelWithSubTopics.name}
       </div>
-      <div className="px-3 py-2 flex flex-col gap-1 md:max-h-[14.3rem] overflow-scroll [&>*:nth-child(1)]:first-alert-row">
+      <div className="px-3 py-2 flex flex-col gap-1 md:max-h-[228px] overflow-scroll [&>*:nth-child(1)]:first-alert-row">
```

Comment:
If it is necessary to use `px`. I think we might want to make a comment to mention what we want to fix.